src/components/Identity/index.js

```diff
@@ -100,6 +146,12 @@ const createIdentity = ({ config, logger, cookie }) => {
     commands: {
       getEcid() {
         return optIn.whenOptedIn().then(getEcid);
+      },
+      setCustomerIDs(options) {
```

Comment:
Have we considered allowing customers to set the customer ids in the config?  Then the setCustomerIDs data exchange would happen on the first request.

Reply:
Yes, I think I will do that in a separate task.
Created a subtask here : https://jira.corp.adobe.com/browse/CORE-32507